src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -105,14 +107,20 @@ def calculate_timeout_seconds(
     timeout_default=pytest.options.timeout_default,
     timeout_maximum=pytest.options.timeout_maximum,
   )
+
+  global_options = options_bootstrapper.bootstrap_options.for_global_scope()
+  colors = global_options.colors
+  env = {"PYTEST_ADDOPTS": f"--color={'yes' if colors else 'no'}"}
```

Comment:
Can you please add a comment explaining why we go out of our way to configure this? Normally I wouldn't expect Pants to need to configure something like this and my instinct would be to try to remove this.